src/App.css

```diff
@@ -55,3 +57,37 @@ tr {
 .card {
   width: 18rem;
 }
+
+.card-container {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 100px;
+}
+
+.card-body {
+  flex-direction: column;
+}
+
+img {
+  height: 200px;
+}
+
+.footer-container {
+  display: flex;
+  background-color: #222;
+  color: white;
+}
+
+ul {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  list-style-type: none;
+}
+
+.selected {
+  background-color: rgba(76, 85, 82, 0.514) !important;
+  color: white;
+  font-weight: bold;
+}
```

Comment:
Highlighting looks good in the table, in this case using !important works fine, but be careful using it in the future, sometime it can cause issues down the line, check out this article to find out more about when to use it: https://css-tricks.com/when-using-important-is-the-right-choice/